tools/cvm-image-rewriter/start-virt.sh

```diff
@@ -11,6 +11,8 @@ GUEST_NAME="tdx-guest"
 GUEST_ROOTDIR=/tmp/libvirt-vms
```

Comment:
Can we make this part of the `args` like `-v vcpu_num` ? othewise if different users run this command, it will conflict

Reply:
Hi Iris, as long as the "GUEST_NAME" (can be customized using arg -n) is not the same, the "GUEST_ROOTDIR" doesn't affect TD boot. If you want to boot more than 1 TD using start-virt.sh, try below commands.
$ ./start-virt.sh -i <guest image> -n <libvirt domain name>
e.g.
$ ./start-virt.sh -i tdx-guest-ubuntu-22.04.qcow2 -n tdx-vm1
$ ./start-virt.sh -i tdx-guest-ubuntu-22.04.qcow2 -n tdx-vm2

Reply:
Thanks @ruomengh  LGTM